src/test/Sentry.Unity.Tests/PlayModeTests.cs

```diff
@@ -157,23 +135,15 @@ private static IEnumerator SetupSceneCoroutine(string sceneName)
             LogAssert.ignoreFailingMessages = true;
         }
 
-        /*
-         * TODO:
-         *
-         * The current Sentry initialization is static. It means that the initialization is done once for all the tests.
-         * That's why we need to alter some state on 'per test' level before running them in bulk.
-         *
-         * This problem will be mitigated as we implement this https://github.com/getsentry/sentry-unity/issues/66
-         */
-        private static TestEventCapture CreateAndSetupSentryTestService()
-        {
-            var testEventCapture = new TestEventCapture();
-            SentryInitialization.EventCapture = testEventCapture;
-            SentryInitialization.ErrorTimeDebounce = new(TimeSpan.FromSeconds(1));
-            SentryInitialization.LogTimeDebounce = new(TimeSpan.FromSeconds(1));
-            SentryInitialization.WarningTimeDebounce = new(TimeSpan.FromSeconds(1));
-            return testEventCapture;
-        }
+        private static void InitSentrySdk(Action<SentryUnity>? unityConfigure = null)
+            => SentryUnity.Init(
+                opt =>
+                {
+                    opt.Enabled = true;
+                    opt.Dsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417";
+                    opt.Logger = new UnityLogger(SentryLevel.Warning);
+                },
+                unityConfigure);

```

Comment:
Snippets are usually formatted like:
```suggestion
            => SentryUnity.Init(o =>
                {
                    o.Enabled = true; // Do we need this? Should be true by default.
                    o.Dsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417";
                    o.Logger = new UnityLogger(SentryLevel.Warning);
                },
                unityConfigure);

```

Alternatively use the full `option` as opposed to `o` though it's far from uncommon the single letter variable name for the lambda argument and keeps the snippet short.